config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
Having already used environment variables in real-world scenarios, I'm going to avoid doing this. It's easy enough to just remove the line in case you *do* mess up your environment, and the naming schema is compatible with how other frameworks do this.